requests/utils.py

```diff
@@ -358,13 +358,12 @@ def get_encoding_from_headers(headers):
 
 def stream_decode_response_unicode(iterator, r):
     """Stream decodes a iterator."""
+    encoding = r.encoding
 
     if r.encoding is None:
```

Comment:
Are you referring to the reuse of "r.encoding" on line 362? I was just trying to follow the pattern established with the [`text()`](https://github.com/kennethreitz/requests/blob/49b69c4751689df7aca4435ef1223a58871a64e2/requests/models.py#L770) function